rai/node/wallet.cpp

```diff
@@ -465,7 +465,7 @@ rai::key_type rai::wallet_store::key_type (rai::wallet_value const & value_a)
 	}
 	else
 	{
-		if ((number >> 32).convert_to<uint32_t> () == 1)
+		if ((number >> 64).convert_to<uint64_t> () == 1)
```

Comment:
Are you sure that this change is correct? I think this would require a database upgrade.

Reply:
@PlasmaPower No, the PR is a bit premature. I wanted to run Travis tests against it.